ui/src/shared/components/Dygraph.js

```diff
@@ -298,6 +297,7 @@ export default class Dygraph extends Component {
   resize = () => {
     this.dygraph.resizeElements_()
     this.dygraph.predraw_()
+    this.dygraph.resize()
```

Comment:
Is there a reason these were decoupled before?

Reply:
It was an oversight not to put them together originally.